From 475567141493feb7fc5fa8704546e2ab1d028219 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Wed, 30 Mar 2016 16:19:24 +0800
Subject: [PATCH] ARM64: dts: rockchip: enable the gt9xx touchscreen on
 rk3399-tb dts

The gt9xx touchscreen should be used by rk3399 evb board.
The rk3399-monkey.dtsi and rk3399-chrome.dtsi just run the different OS.
So... we should move the touchscreen node into the rk3399-tb.dtsi.

Change-Id: Ida8203e045e0fc0eb49e8a37e4ad609c230e040f
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-monkey.dts | 4 ----
 arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi    | 1 -
 2 files changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts b/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
index aeca00141ed1..c38c6ed8cf3a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-monkey.dts
@@ -130,10 +130,6 @@
 	};
 };
 
-&gt9xx {
-	status = "okay";
-};
-
 &fb {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi
index 88c25a4b8a42..79cd31260004 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tb.dtsi
@@ -460,7 +460,6 @@
 		max-y = <1900>;
 		tp-size = <911>;
 		tp-supply = <&vcc3v0_tp>;
-		status = "disabled";
 	};
 };
 
-- 
2.35.3

